samcli/commands/_utils/options.py

```diff
@@ -788,3 +789,21 @@ def _space_separated_list_func_type(value):
 
 
 _space_separated_list_func_type.__name__ = "LIST"
+
+
+def generate_next_command_recommendation(command_tuples: List[tuple]):
+    """
+    Generates a message containing some suggested commands to run next.
+
+    :type command_tuples: list
+    :param command_tuples: list of tuples containing the command with their respective description
+    """
+    template = """
+Commands you can use next
+=========================
+{}"""
+    command_list_txt = ""
+    for description, command in command_tuples:
+        command_list_txt += f"[*] {description}: {command}\n"
+    command_list_txt.rstrip()
```

Comment:
You simplify and avoid the `rstrip()` by joining values from a list comprehension (the inline list definition) instead:

```suggestion
    command_list_txt = "\n".join(f"[*] {description}: {command}" for description, command in command_tuples)
```

Reply:
Thank you for the suggestion, made changes to implement this